res/css/views/dialogs/_DevtoolsDialog.scss

```diff
@@ -14,8 +14,18 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+.mx_DevTools_content {
+    margin: 10px 0;
+}
+
 .mx_DevTools_RoomStateExplorer_button, .mx_DevTools_RoomStateExplorer_query {
     margin-bottom: 10px;
+    max-width: 684px;
```

Comment:
What's this "magic" value?

Reply:
The dialog is hard-coded to a width of 704px (I've no idea why), subtract the margins and padding on the dialog (that can't be changed from this component, or without messing with a lot of other stuff) and you get/got 684px.

However,  this was when I was attempting to wrap all of the devTools content in "mx_DevTools_content" containers for consistency, and it turns out that this is a lot bigger job than I was hoping for. Without making those changes this doesn't actually make a lot of sense, so I'll back it out. Thanks for catching that.